system/t12_api/publish.py

```diff
@@ -3,6 +3,11 @@
 
 from api_lib import TASK_SUCCEEDED, APITest
 
+try:
+    import requests
+except ImportError:
+    requests = None
+
```

Comment:
Why is this needed? I think this should be removed

Reply:
You are right, I forgot to remove this...